### ISSUE TITLE
Support alias paths

### ISSUE THREAD

**Issue by tomahl** [mayank99/ecsstatic#8]
Hey,

Trying out this awesome project and noticed that variables imported with custom aliases does not work:

`[plugin:ecsstatic] Unable to resolve expression in template literal.`

```tsx
import { COLORS } from '@constants/colors'; // <-- Does not work
import { COLORS } from '../../../constants/colors'; // <-- Works

const fooBar = css`
    color: ${COLORS.red}
`
```
tsconfig.json
```json
{
	"compilerOptions": {
		"paths": {
			"@constants/*": ["./src/constants/*"]
		}
	}
}
```

vite.config.ts
```ts
export default defineConfig({
	resolve: {
		alias: {
			'@constants': path.resolve(__dirname, './src/constants'),
		},
	},
	plugins: [ecsstatic()],
});
```

Thanks for your hard work! 👍

**Comment by mayank99** [comment]
This is interesting, I would have expected vite to resolve those aliases before executing the plugin. Will look into it, thanks

**Comment by mayank99** [comment]
Fixed in 0.5.1 🎉
example: https://stackblitz.com/edit/github-uier57?file=src%2Fconstants%2Fcolors.ts,src%2FLogo.tsx

**Comment by tomahl** [comment]
Great, thanks 👍
